tsconfig.json

```diff
@@ -5,6 +5,9 @@
     "paths": {
       "@defichain/salmon-*": [
         "packages/salmon-*/src"
+      ],
+      "@defichain/salmon": [
```

Comment:
to reverting this change

Reply:
ok I think i got it

Reply:
I managed to revert it to /main but github is still showing this change (although it says outdated here on the comments)